src/taipy/core/taipy.py

```diff
@@ -463,3 +464,40 @@ def clean_all_entities() -> bool:
     _CycleManagerFactory._build_manager()._delete_all()
     _JobManagerFactory._build_manager()._delete_all()
     return True
+
+
+def export_scenario(
```

Comment:
```suggestion
def export_scenario(scenario: Union[Scenario, ScenarioId], **kwargs):
```

Instead of just the id, we can expand this further and accept the entity object itself too. What kind of formats we're expecting to export, if we're only expecting to export them as Json, wouldn't `folder_path` be better than `kwargs`?

Do we also want to expand this feature to other entities like Cycle, Pipeline, Task?

Reply:
We only plan to export as JSON files. 

In this ticket, we are only considering exporting scenarios. We may want to expand the feature to other entities in the future. But not right now.